devkit/sandbox/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # @rpch/sandbox
 
+## 0.2.0
```

Comment:
please use 

`yarn changeset add`

instead of editing this file manually

Reply:
In the future or for this PR?

Reply:
its easy. best do it for this PR